### PR TITLE
[2.13.x] DDF-4008 Update jetty to 9.3.24.v20180605

### PR DESCRIPTION
**Pull request by kcover**
#### What does this PR do?

Updates jetty to 9.3.24.v20180605

#### Who is reviewing it? 

@peterhuffer @emmberk @oconnormi @brjeter

#### Ask 2 committers to review/merge the PR and tag them here.

@clockard
@stustison

#### How should this be tested?

install ddf and execute "feature:info pax-jetty" in the console and verify the bundles are using the 9.3.24.v20180605 version. Make sure that you can log in and travel to the admin and intrigue UIs.

#### What are the relevant tickets?

[DDF-4008](https://codice.atlassian.net/browse/DDF-4008)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: features/kernel/src/main/feature/feature.xml [hunk 1]
```diff
@@ -20,6 +20,59 @@
...
+            <bundle start-level="30">mvn:org.eclipse.jetty.websocket/websocket-client/${jetty.version}</bundle>
+            <bundle start-level="30">mvn:org.eclipse.jetty.websocket/websocket-common/${jetty.version}</bundle>
+            <bundle start-level="30">mvn:org.eclipse.jetty.websocket/websocket-servlet/${jetty.version}</bundle>
+            <bundle start-level="30">mvn:org.eclipse.jetty.websocket/websocket-api/${jetty.version}</bundle>
+            <bundle start-level="30">mvn:org.eclipse.jetty.websocket/javax-websocket-server-impl/${jetty.version}</bundle>
+            <bundle start-level="30">mvn:org.eclipse.jetty.websocket/javax-websocket-client-impl/${jetty.version}</bundle>
+            <bundle start-level="30">mvn:javax.websocket/javax.websocket-api/1.1</bundle>
+        </conditional>
+
+
```
Comment (emmberk) [inline features/kernel/src/main/feature/feature.xml]: ✏️ Extra line

File: features/kernel/src/main/feature/feature.xml [hunk 2]
```diff
@@ -20,6 +20,59 @@
     <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
     <repository>mvn:io.fabric8/fabric8-karaf-features/${fabric8.version}/xml/features</repository>
 
+    <!-- This feature replaces the jetty version provided by karaf, which has been disabled in blacklisted.properties -->
```
Comment (emmberk) [inline features/kernel/src/main/feature/feature.xml]: ✏️ Maybe add a note here and in blacklisted.properties about when these lines can be removed (i.e. when DDF is updated to a Karaf version that has a good jetty version)

File: distribution/ddf-common/src/main/resources/etc/blacklisted.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+pax-jetty;type=feature;range=[9.3,9.3.21]
```
Comment (clockard) [inline distribution/ddf-common/src/main/resources/etc/blacklisted.properties]: Could we add 
jetty;type=feature
pax-jetty-http2;type=feature
to clean up uninstalled jetty features as well?

**Comment by kcover** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by kcover** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**